modules/apps/object/object-web/src/main/resources/META-INF/resources/js/index.d.ts

```diff
@@ -52,7 +51,7 @@ type ObjectFieldSettingName =
 interface ObjectValidation {
 	active: boolean;
 	description?: string;
-	engine: ObjectValidationType;
+	engine: string;
```

Comment:
@cgmonte @MariMalta Not about the code itself, but I think this commits message should be something like "Update interfaces".